lib/media/presentation_timeline.js

```diff
@@ -566,3 +608,21 @@ shaka.media.PresentationTimeline = class {
     }
   }
 };
+
+/**
+ * @typedef {{
+ *   start: number,
+ *   end: number
+ * }}
+ * @export
```

Comment:
Why is it necessary?

Reply:
It's used in a public method of PresentationTimeline which is exported.